test/elm/comparison/comparison-test.ts

```diff
@@ -71,20 +71,20 @@ describe('Equal', () => {
     this.impossiblyEqualDateTimes.exec(this.ctx).should.be.false();
   });
 
-  it('should be null for Date and DateTime equality with same year, month, hour', function () {
-    should(this.dateAndDateTimeNull.exec(this.ctx)).be.null();
+  it('should be true for Date and DateTime equality with same year, month, hour', function () {
+    this.dateAndDateTimeNull.exec(this.ctx).should.be.true();
```

Comment:
The name of the CQL expression (and variable here) is now out of sync w/ the expectation.  Now that this returns `true`, the CQL expression should be `DateAndDateTimeTrue` (and corresponding variable here should be `dateAndDateTimeTrue`).

All of the test expressions and variables need to be checked and fixed to ensure their name still matches the expectation. (So I won't comment on anymore like this).